Sombra.Web/wwwroot/css/main.css

```diff
@@ -621,7 +621,7 @@ footer .footer-info-right {
     float: right;
 }
 
-footer .footer-info-right .footer-youtube, .footer-twitter, .footer-instagram, .footer-facebook {
```

Comment:
lets try and change main only in 1 branch (like home). Otherwise having branches dont make sense